scripts/us_fema/national_risk_index/generate_schema_and_tmcf.py

```diff
@@ -215,15 +217,14 @@ def normalize_unit_for_measured_property(measured_property, unit):
     Given a measured_property and unit, normalize it to KG terms.
     If m_prop is Expected Loss, makes the unit be USD if it is not Score.
```

Comment:
Does comment need update?

Reply:
it was logically equivalent, but applied the same refactoring to the comment to make it simpler